package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-image-gallery",
-  "version": "0.8.23-sl",
+  "version": "0.8.25-sl",
```

Comment:
Bumped this to x.x.25, because we already have a tag for version x.x.24, so it would've been a bit misleading.